packages/harpguru-core/src/components/menu/menu.tsx

```diff
@@ -0,0 +1,68 @@
+import Animated from 'react-native-reanimated'
+import { StyleSheet } from 'react-native'
+import React from 'react'
+
+import { getMenuStylesAndAnimationVals } from '../../utils'
+import { MenuProps } from '../../types'
+import { getSizes } from '../../styles'
+
+type LocalMenuProps = MenuProps & {
+  readonly children: React.ReactNode
+}
+export const Menu = ({
```

Comment:
Should this be called `MenuWrapper` or `MenuAnimation` or `MenuBase` or something?

Reply:
Nah. It's the top level component wherever it's implemented. I think it's clear enough what it is.